readme.md

```diff
@@ -42,6 +42,7 @@
 - [npm-compare](https://npmcompare.com) - Easily search and compare packages.
 - [npm-top](https://gist.github.com/bcoe/dcc961b869bbf6685002) - npm users by downloads.
 - [npm semver calculator](http://semver.npmjs.com) - Visually explore what versions of a package a semver range matches.
+- [npm-stats](http://www.npm-stats.com/) - NPN stats displays metrics about npm modules
```

Comment:
You have multiple typos here and no point in mentioning the name in the description.

`modules` => `packages`

and you have to end it with a `.`